include/tiramisu/core.h

```diff
@@ -2483,7 +2482,38 @@ class computation
       */
     computation(std::string name,std::vector<var> iterator_variables, tiramisu::expr e, bool schedule_this_computation, primitive_t t);
 
+    /**
+      * \brief Construct the distribution map of a computation.
+      * Not safe to use. It's currently being implemented.
+      *
+      * A distribution map is a map that describes how the computation is distributed across ranks.
```

Comment:
Can you also say that this map "partitions" the computation? (the term partitioning is well known so we should mention it).